Formula/mingw-w64.rb

```diff
@@ -117,6 +118,8 @@ def install
       end
 
       mkdir "mingw-w64-crt/build-#{arch}" do
+        # Resolves "Too many open files" binutils-bfd issue.
+        ENV.deparallelize
```

Comment:
Has this been reported upstream? Do we have to deparallelize everything, or can we just do it for specific steps?
e.g.
```ruby
ENV.deparallelize { system "make" }
```
just after invoking `configure`.

Reply:
The issue I’d found for binutils was closed marked as resolved but it still happens.

Only the crt needs to be ran deparallelize

Reply:
> The issue I’d found for binutils was closed marked as resolved but it still happens.

Add a link to this in a comment before doing `ENV.deparallelize`, please. And if you don't mind posting a follow-up to mention that it's still happening, that'd be great.

> Only the crt needs to be ran deparallelize

Let's just do it for those steps though. If you need to deparallelize multiple commands:
```ruby
ENV.deparallelize do
  system "foo"
  system "bar"
end
```

Reply:
I’ll revise this proper once I’m home.

Reply:
I’m assuming swapping to draft will low me to still make edits but the Ci won’t be ran?

Reply:
CI is still run on drafts. You can add a `system "false"` call at the start of the `install` method to get CI to fail quickly. (I can also kill the tests, but then I'll need to do that every time you push something.)

Just don't forget to remove the `system "false"` when you're done.

Reply:
I didn't receive the registration email still not even in spam to report the issue.

<br>

Here is the error I get when compiling `i686-w64-mingw32-crt` `-j8`
```
:info:build i686-w64-mingw32-dlltool: bfd_open failed open stub file: dfxvs01181.o: Too many open files in system
:info:build i686-w64-mingw32-dlltool: i686-w64-mingw32-dlltool: bfd_open failed open stub file: dvxvs00563.o: Too many open files in systembfd_open
```
Here are too similar Bug reports both related to bfd;\
- https://sourceware.org/bugzilla/show_bug.cgi?id=24723
- https://sourceware.org/bugzilla/show_bug.cgi?id=23573

So no sure if I should make this as resolved or not considering, but I did add this information directly into the formula.